examples/website/i3s/app.js

```diff
@@ -178,20 +190,21 @@ export default class App extends PureComponent {
       loadOptions.i3s = {token};
     }
     const layerType = metadata?.layers[0]?.layerType;
-    return layerUrls.map((url, index) => (
-      new Tile3DLayer({
-        id: `tile-layer-${index}`,
-        data: url,
-        loader: I3SLoader,
-        onTilesetLoad: this._onTilesetLoad.bind(this),
-        onTileLoad: () => this._updateStatWidgets(),
-        onTileUnload: () => this._updateStatWidgets(),
-        // TODO enable it when Building Scene Layer picking will be implemented.
-        pickable: layerType !== BUILDING_SCENE_LAYER_TYPE,
-        loadOptions,
-        highlightedObjectIndex: selectedFeatureIndex
-      })
-    ));
+    return layerUrls.map(
```

Comment:
Each sublayer is an independent layer with type "3DObjects".

Reply:
Yes I understand. At least one concerns is that they all have their own request schedulers. The job of the request scheduler is partially to make sure we don't issue more requests than the browser can process in parallel. This is defeated when we don't share a scheduler between all these tilesets. It may impact performance when panning and zooming as a lot more requests that are no longer needed will be queued up.

Reply:
I see. Now data flows though deck.gl. The tileset object (with scheduler) is created in deck.gl. We need to think about all this interaction between loaders.gl and deck.gl